docs/protocol_overview.md

```diff
@@ -183,15 +183,15 @@ The leader has **authority**.
 - The leader can choose which agenda to include in the block if there are multiple eligible agendas.
 - The leader can choose which chat chain to semifinalize.
 
-A *good* node should be objective to agendas; they should choose the first eligible agenda, instead of waiting until it observes another one in their personal favor. Similarly, a 'good' node should be objective to the chat chain semifinalization as well.
+A *good* node should be objective to agendas; they should choose the first eligible agenda, instead of waiting until it observes another one in their personal favor. Similarly, a *good* node should be objective to the chat chain semifinalization as well.
 
 As long as the selected first few leaders are honest, faithful, and even good,
 the network will be not only **live** but also **fair**. (note that it's always **safe** regardless of the leader's behavior; safety is guaranteed by the BFT consensus)
 
 ### Single Point of Failure
 
 To summarize, the leader has **responsibilities** and **authority**.
-Actually, this is not so different from other blockchains that use a leader-based BFT consensus. The leader should be online for the whole round, to receive enough amount of transactions (in the 'mempool'), and the leader chooses which transactions to include in the block. The only difference is that we don't have a user-signed transaction but a governance-signed agenda instead and that we have an additional protocol for the chat finalization.
+Actually, this is not so different from other blockchains that use a leader-based BFT consensus. The leader should be online for the whole round, to receive enough amount of transactions (in the 'mempool'), and the leader chooses which agenda to include in the block. The only difference is that we don't have a user-signed transaction but a governance-signed agenda instead and that we have an additional protocol for the chat finalization.
```

Comment:
Here we refer to **transactions**, because we're comparing other conventional blockchains such as Ethereum, Cosmos, ..
The equivalent of an 'agenda' in our protocol is a 'transaction' in Ethereum. (The unit of inclusion, The unit of cryptographic proof)

Reply:
@junha1 Yeah, I understood, but I'm not sure which one is more appropriate for this sentence. If you're saying that 'transaction' is more appropriate, I will update it as you requested.

Also, it could be better more clearly describe the agenda and transaction with some diagrams in the beginning of the document. Thanks!

Reply:
In our protocol, transaction is merely a unit of state transition. **Nobody signs on a single transaction, but on an agenda which's a sequence of txes**.

Reply:
@kjhman21 Let's stick to 'transaction' and let me elaborate the description further in the next work, using a figure as you suggested.

Reply:
@junha1 Of course. I will update that soon!

Reply:
@junha1 rollbacked!!